shared/common-adapters/avatar.desktop.js

```diff
@@ -20,6 +20,15 @@ class Avatar extends Component<void, Props, State> {
     this.state = {avatarLoaded: false, errored: false}
   }
 
+  componentWillReceiveProps (nextProps: Props) {
+    const url = shared.createAvatarUrl(this.props) || noAvatar
```

Comment:
No need to use the `|| noAvatar` because they'll both return null if that's the case, and `null === null` is `true`